starknet_devnet/server.py

```diff
@@ -123,22 +92,17 @@ def load_config(self):
     def load(self):
         return self.application
 
-
 def main():
     """Runs the server."""
 
-    args = parse_args()
-
     # Uncomment this once fork support is added
     # origin = Origin(args.fork) if args.fork else NullOrigin()
     # starknet_wrapper.origin = origin
 
+    args = parse_args(sys.argv[1:])
+    state.set_starknet_wrapper(StarknetWrapper(DevnetConfig(args)))
```

Comment:
It is called two times when you start an app on state init and after that here again.

Reply:
Doesn't affect performance, tests using app.test_client don't work without it.